api/app/src/sequelize/migrations/20221223020000-alter-connected_user-add-index.ts

```diff
@@ -0,0 +1,20 @@
+import type { Migration } from "..";
+import { ConnectedUser } from "../../models/connected-user";
+
+// Use 'Promise.all' when changes are independent of each other
+// Docs: https://sequelize.org/api/v6/class/src/dialects/abstract/query-interface.js~queryinterface
+export const up: Migration = async ({ context: queryInterface }) => {
+  return queryInterface.sequelize.transaction(async (transaction) => {
+    await queryInterface.addIndex(ConnectedUser.NAME, ["cx_id"], {
+      transaction,
+    });
```

Comment:
Realized we don't have an index on `cxId` on this table, that's likely the most used search key we'll be using.